### DISCUSSION TITLE
TSML UI Region Drop Down Question

### DISCUSSION THREAD

**Discussion by steve970** [code4recovery/12-step-meeting-list#1566] [Q&A] [answered]
Hello, I am currently using the TSML UI to add my meetings and would like to make the dropdown for the filter to be collapsable and not have the count next to each region and subregion. How do I go about doing that? Thanks.

**Comment by joshreisner** [comment, marked as answer]
we do have an issue for that which you can follow and chime in on: https://github.com/code4recovery/tsml-ui/issues/411 - we don't have an ETA on that or a developer assigned to take on the work yet

**Comment by steve970** [comment]
I added to the the conversation from the link you sent me. Thank you.
